bin/improver-spot-extract

```diff
@@ -84,6 +88,22 @@ def main():
         " in altitude to the spot site within the search radius defined when"
         " the neighbour cube was created. May be used with land_constraint.")
 
+    percentile_group = parser.add_argument_group(
+        title="Extract a percentile",
+        description="Extract a particular percentile from probabilistic "
+        "data.")
+    percentile_group.add_argument(
+        "--extract_percentile", default=None,
```

Comment:
I think this needs to say it will do this for percentile and probability input (and realizations? -see comment below) and if no probabilistic dimension is found then a warning is raised

Reply:
I've added more to the group description.